services/QuillLMS/client/app/bundles/Teacher/containers/StudentProfile.jsx

```diff
@@ -29,11 +29,8 @@ class StudentProfile extends React.Component {
     if (classroomId) {
       handleClassroomClick(classroomId);
       fetchStudentProfile(classroomId)
```

Comment:
Do we know if this `fetchStudentProfile` call in cases where a `classroomId` is passed in is used?  It's probably not worth a deep-dive investigation since I suspect that the fix you have here resolves most calls, but I wonder if we can eliminate this entirely.

Reply:
Yeah, that's something we could look in to in the future